client/signup/config/steps-pure.js

```diff
@@ -405,6 +405,18 @@ export function generateSteps( {
 			},
 		},
 
+		'plans-affiliate': {
+			stepName: 'plans-affiliate',
+			apiRequestFunction: addPlanToCart,
+			dependencies: [ 'siteSlug' ],
+			optionalDependencies: [ 'emailItem', 'themeSlugWithRepo' ],
```

Comment:
@southp you are probably the expert here. Are these `optionalDependencies` necessary and correct?

Reply:
I see. Thanks for taking the time to explain.